scripts/common/actor.js

```diff
@@ -224,6 +224,70 @@ export class AgeOfSigmarActor extends Actor {
         this.power.isUndercharge =             this.power.consumed > this.power.capacity;
     }
 
+    /**
+     * applies Damage to the actor
+     * @param {int} damages 
+     */
+    async applyDamage(damage) {
+        let remaining = this.combat.health.toughness.value - damage;
+         // Update the Actor
+         const updates = {
+            "data.combat.health.toughness.value": remaining >= 0 ? remaining : 0
+        };
+
+        // Delegate damage application to a hook
+        const allowed = Hooks.call("modifyTokenAttribute", {
+            attribute: "combat.health.toughness.value",
+            value: this.combat.health.toughness.value,
+            isDelta: false,
+            isBar: true
+        }, updates);
+
+        let ret = allowed !== false ? this.update(updates) : this;
+
+        // Doing this here because foundry throws an error if wounds are added before the update
+        if(remaining < 0 && this.combat.health.wounds.max > 0) {          
+            this.addWound(remaining);
+        }
+        return ret;
```

Comment:
I think it'd be useful to also include a notification for damage application. As in D&D you don't really get feedback when you click to apply damage. 

Perhaps add 
```js
ui.notifications.notify(`${damage} Damage applied to ${this.data.token.name}`)
```

Or if you want it visible to everyone, create a chat message.

Reply:
I added a notification to the apply damage method instead of a template string a used a localized one